routes/weatherProviders/DarkSky.ts

```diff
@@ -105,9 +107,61 @@ async function getDarkSkyWeatherData( coordinates: GeoCoordinates ): Promise< We
 	return weather;
 }
 
+async function getDarkSkyEToData( coordinates: GeoCoordinates ): Promise< EToData > {
+	// TODO use a rolling 24 hour window instead of fixed calendar days?
+	// The Unix epoch seconds timestamp of 24 hours ago.
+	const timestamp: number = moment().subtract( 1, "day" ).unix();
+
+	const DARKSKY_API_KEY = process.env.DARKSKY_API_KEY,
+		historicUrl = `https://api.darksky.net/forecast/${DARKSKY_API_KEY}/${coordinates[0]},${coordinates[1]},${timestamp}?units=si`;
```

Comment:
Elsewhere we use Imperial units. I think we should standardise on a units system for Weather Providers and convert if needed in the Adjustment Method (e.g. ETo calculation).

Reply:
Good idea, will do

Reply:
Resolved in c8b948b